task/stat_test.go

```diff
@@ -0,0 +1,35 @@
+package task
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/yunify/qsctl/v2/pkg/types"
+	"github.com/yunify/qsctl/v2/utils"
+)
+
+func TestNewStatTask(t *testing.T) {
+	cases := []struct {
+		input            string
+		expectedTodoFunc types.TodoFunc
+		expectErr        error
+	}{
+		{"qs://test-bucket/obj", NewObjectStatTask, nil},
+		{"qs://test-bucket/obj/", NewObjectStatTask, nil},
```

Comment:
`obj/` end with `/` should be treated as a pseudo dir in remote.

Reply:
But there is no difference between stating an object and a pseudo dir, both of them  got the same behavior.

Reply:
Well, I think a NewPseudoDirStatTask is more unambiguous, let's leave a comment here and return error for this saturation.

Reply:
OK，I'll make another commit to handle this.